app/controllers/concerns/foreman/controller/discovered_extensions.rb

```diff
@@ -70,19 +70,23 @@ def perform_reboot_all hosts = Host::Discovered.all
     result = true
     error_message = _("Errors during reboot: %s")
     overall_errors = ""
+    host_details  = []
 
     if hosts.count > 0
       hosts.each do |discovered_host|
         begin
           unless discovered_host.reboot
             error =  "#{discovered_host.name}: failed to reboot "
             overall_errors << error
-            logger.error error
```

Comment:
You are effectively deleting logging of the exception, we want to keep all the errors in log for each individual host which failed to be able to debug problems, Please keep it, or use Foreman exception logger helper method to format it properly.